webapps/frontend/app/views/site/vdisks.html

```diff
@@ -65,7 +65,8 @@ <h2 class="pull-left" data-i18n="ovs:vdisks.title"></h2>
                 <td>
                     <div data-bind="status: {
                                         colors: {
-                                            green: ['ok_standalone', 'ok_sync'].contains(dtlStatus()),
+                                            blue: dtlStatus() == 'ok_standalone',
```

Comment:
Maybe it's better to keep it green if manual standalone or if standalone and only one possible target, and orange if standalone but multiple targets exist.